Source/Alamofire.swift

```diff
@@ -27,42 +27,83 @@ import Foundation
 /// Types adopting the `URLStringConvertible` protocol can be used to construct URL strings, which are then used to
 /// construct URL requests.
 public protocol URLStringConvertible {
-    /// A URL that conforms to RFC 2396.
+    /// A URL string that conforms to RFC 2396.
     ///
     /// Methods accepting a `URLStringConvertible` type parameter parse it according to RFCs 1738 and 1808.
     ///
     /// See https://tools.ietf.org/html/rfc2396
     /// See https://tools.ietf.org/html/rfc1738
     /// See https://tools.ietf.org/html/rfc1808
-    var urlString: String { get }
+    var urlString: String? { get }
```

Comment:
I don't think it's necessary to include the property in the protocol definition anymore. Just the protocol extension is enough to provide the property.